src/main/resources/map_2.txt

```diff
@@ -0,0 +1,17 @@
+name = map_1
```

Comment:
I think this file was not intended to be added, please make sure you check what is being added by your commits